test/bigtable/index.js

```diff
@@ -110,6 +110,24 @@ describe('Bigtable', function() {
       fakeUtil.normalizeArguments = normalizeArguments;
     });
 
+    it('should leave the original options unaltered', function() {
```

Comment:
It does that, I'm doing a `for in` loop on the original options and asserting that they're in the options that get passed to `GrpcService`.